.pre-commit-config.yaml

```diff
@@ -16,14 +16,14 @@ repos:
       - id: flake8
 
   - repo: https://github.com/pycqa/isort
-    rev: 5.10.1
+    rev: 5.12.0
     hooks:
       - id: isort
         name: isort (python)
         args: [--settings-path=pyproject.toml]
 
   - repo: https://github.com/pre-commit/mirrors-mypy
-    rev: v0.991
+    rev: v1.0.0
```

Comment:
Are we upgrading black and mypy yet?

Reply:
It seems I am a little too pro-active. We will discuss this at standup tomorrow.